### PR TITLE
[Localization] Menu.ts Spanish translation

### PR DESCRIPTION
**Pull request by Alekemon**
Just localization changes so everything in Menu.ts is translated to Spanish

### CODE REVIEW

File: src/locales/es/menu.ts [hunk 1]
```diff
@@ -36,11 +36,11 @@ export const menu: SimpleTranslationEntries = {
   "boyOrGirl": "¿Eres un chico o una chica?",
   "boy": "Chico",
   "girl": "Chica",
-  "evolving": "What?\n{{pokemonName}} is evolving!",
-  "stoppedEvolving": "{{pokemonName}} stopped evolving.",
-  "pauseEvolutionsQuestion": "Would you like to pause evolutions for {{pokemonName}}?\nEvolutions can be re-enabled from the party screen.",
-  "evolutionsPaused": "Evolutions have been paused for {{pokemonName}}.",
-  "evolutionDone": "Congratulations!\nYour {{pokemonName}} evolved into {{evolvedPokemonName}}!",
+  "evolving": "¿Anda?\n¡{{pokemonName}} está evolucionando!",
```
Comment (flx-sta) [inline src/locales/es/menu.ts:39]: Is the `i` in front of `{{pokemonName}}` on purpose (not a Spanish speaker)

**Review by flx-sta — APPROVED** [review]
approved with comments

**Comment by flx-sta** [comment]
Seems to be a duplicate of #1551

File: src/locales/es/menu.ts (hunk 1 above)
Comment (GoldTra) [reply src/locales/es/menu.ts:39]: That "¡" is an exclamation

**Comment by bennybroseph** [comment]
Please create this using the template

**Pull request closed without merging**